docs/getting-started/install-configure.md

```diff
@@ -174,15 +174,15 @@ curl -sL https://raw.githubusercontent.com/crossplane/crossplane/master/install.
 <div class="tab-pane fade" id="install-tab-cli-latest" markdown="1">
 
 ```console
-curl -sL https://raw.githubusercontent.com/crossplane/crossplane/master/install.sh | CHANNEL=master sh
+CHANNEL=master | curl -sL https://raw.githubusercontent.com/crossplane/crossplane/master/install.sh | sh
```

Comment:
@terrytangyuan based on the PR body, I think the current install command works, you just left of `sh` in your invocation. You have:
```
curl -sL https://raw.githubusercontent.com/crossplane/crossplane/master/install.sh | CHANNEL=master
```
and the command is:
```
curl -sL https://raw.githubusercontent.com/crossplane/crossplane/master/install.sh | CHANNEL=master sh
```

Reply:
Indeed. I should definitely be more cautious when submitting PRs on Friday afternoons. Thanks for taking the time to respond. Closing as change is not needed.